### PR TITLE
<DEVOPS-128> creating biforst service SA

### PR DESCRIPTION
**Pull request by pankajholariyacb**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -715,6 +722,14 @@ variable "customer_service_account_roles" {
 
 }
 
+variable "bifrost_service_account_roles" {
+  description = "Roles for bifrost services"
+  type        = list(string)
+  default     = ["roles/datastore.user","roles/pubsub.editor"]
```
Comment (Sunil-Jacob) [inline variables.tf:728]: @pankajholariyacb @PankaHolariya only give `roles/pubsub.publisher` role

**Review by Sunil-Jacob — CHANGES_REQUESTED** [review]
Added review

File: variables.tf (hunk 1 above)
Comment (pankajholariyacb) [reply variables.tf]: done, now only provided `roles/pubsub.publisher ` role

Comment (Sunil-Jacob) [reply variables.tf]: @pankajholariyacb  i dont see it

Comment (pankajholariyacb) [reply variables.tf]: required changes done sir

**Review by Sunil-Jacob — APPROVED** [review]
LGTM

**Pull request merged**